src/reducers/metadata.js

```diff
@@ -28,6 +28,7 @@ import { requestStates } from "../constants";
 export const ORDER_LOOKUP_MODULE_NAME = "order";
 export const CUSTOMER_LOOKUP_MODULE_NAME = "customer";
 export const PRODUCT_LOOKUP_MODULE_NAME = "product";
+export const CUSTOMER_PRODUCT_DEFINITIONS_MODULE_NAME = "customer";
```

Comment:
CUSTOMER_PRODUCT_DEFINITIONS_MODULE_NAME rename to CUSTOMER_DEFINITIONS_MODULE_NAME (remove Product)